testing/scenario_app/ios/Scenarios/Scenarios/Util.h

```diff
@@ -0,0 +1,12 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+
+@interface Util : NSObject
```

Comment:
Util isn't a great name.  Maybe make this a FlutterEngine category?  Also _Nullable shouldn't be used on NSObjects.

```objective-c
NS_BEGIN_ASSUME_NONNULL
@interface FlutterEngine (ScenarioTesting)
- (void)runWithScenario:(NSString*)scenario
                                   withCompletion:(void (^_Nullable)(void))engineRunCompletion;
@end

```

Reply:
oh ya, good idea. Done. 

> Also _Nullable shouldn't be used on NSObjects.

Not sure I understood. But I think I did what you meant.

Reply:
Sorry, I meant to use `nullable` instead of `_Nullable`.  Rule of thumb if the compiler lets you use  `nullable`, then use it.

Reply:
ohhhh `withCompletion:(void (^_Nullable)(void))engineRunCompletion;` should be `withCompletion:(nullable void (^)(void))engineRunCompletion;`. Sigh block syntax...